Sources/RequestKit/JSONPostRouter.swift

```diff
@@ -5,12 +5,15 @@ import FoundationNetworking
 
 public protocol JSONPostRouter: Router {
     func postJSON<T>(_ session: RequestKitURLSession, expectedResultType: T.Type, completion: @escaping (_ json: T?, _ error: Error?) -> Void) -> URLSessionDataTaskProtocol?
+    func post<T: Codable>(_ session: RequestKitURLSession, dateDecodingStrategy: JSONDecoder.DateDecodingStrategy?, expectedResultType: T.Type, completion: @escaping (_ json: T?, _ error: Error?) -> Void) -> URLSessionDataTaskProtocol?
```

Comment:
Given T is the item that is decoded, should T be just Decodable (even in the older functions)?

Reply:
Agree, but considering the scope of influence I create another [PR](https://github.com/nerdishbynature/RequestKit/pull/97).